tools/src/scie_pants/pants_version.py

```diff
@@ -99,13 +102,21 @@ def determine_tag_version(
     stable_version = Version(pants_version)
     if stable_version >= PANTS_PEX_GITHUB_RELEASE_VERSION:
         return ResolveInfo(stable_version, sha_version=None, find_links=None)
+    elif stable_version >= PANTS_GITHUB_WHEELS_START_VERSION:
+        return ResolveInfo(
+            stable_version,
+            sha_version=None,
+            find_links="https://wheels.pantsbuild.org/simple/",
```

Comment:
This is broken since this new bucket doesn't contain all the 3rdparty pre-built wheels in addition. This is required for Mac Arm support since there are basically never pre-built Mac Arm native wheels, Pants uses those, and PBS can't build native wheels.